src/main/java/helpers/Constant.java

```diff
@@ -8,4 +8,6 @@
     public static final int DEFAULT_PASSWORD_LENGTH = 8;
     public static final int DEFAULT_PID_LENGTH = 8;
     public static final String DEFAULT_DATE_FORMAT = "M/d/yyyy";
```

Comment:
should be MM/dd/yyyy

Reply:
The format is for ticket booking testcases (which I haven't used yet). And on the Book ticket page, the date format is "M/d/yyyy", that's why I use it.